src/gmpints.c

```diff
@@ -297,6 +318,38 @@ Obj GMP_REDUCE( Obj gmp )
   return gmp;
 }
 
+#if DEBUG_GMP
```

Comment:
I am not sure about using IS_NORMALIZED_AND_REDUCED from packages (ideally, packages should never assume that they know how GAP integers work internally; I'd prefer to add APIs that do whatever packages need... but of course that just an idealistic wish, and not realistic right now).

In any case, a strong reason why we should indeed always compile `IS_NORMALIZED_AND_REDUCED` is to preven bitrot in it. I'll make that change.